views/admin/settings/standard.html.twig

```diff
@@ -3,9 +3,475 @@
     <header class="jumbotron subhead" id="overview">
         <h1><i class="glyph2x-adjust"></i>Baïkal settings</h1>
     </header>
-    <form>
+    <form class="form-horizontal">
+    	<input type="hidden" name="refreshed" value="0">
         <fieldset>
             <legend style="line-height: 40px;">Editing Standard<i class="glyph-cogwheel"></i><strong>Baïkal Settings</strong></legend>
+            <div class="control-group">
+				<label class="control-label" for="PROJECT_TIMEZONE">Server Time zone</label>
+				<div class="controls">
+					<select class="" id="PROJECT_TIMEZONE" name="data[PROJECT_TIMEZONE]">
```

Comment:
http://php.net/manual/en/function.timezone-identifiers-list.php 
It might be nice to make this list dynamic :)